papper/relatorio.tex

```diff
@@ -269,13 +271,26 @@ \subsection{UC08 -- Remover um Status}
 \end{enumerate}
 
 
-\section{Conclus�o}
+\section{Tecnologias utilizadas}
 
-Portanto temos como finalidade, prover um sistema �gil capaz de disponibilizar
-r�pidos feedbacks para a comunidade, e pela comunidade. Com a informa��o � m�o de forma mais veloz
-, a tomada de decis�o dos frenquentadores em rela��o ao RU ser� potencializada, permitindo uma
-economia de tempo, comodidade, al�m de possuir dados reais que espelhem a realidade vivenciada
-pelos estudantes, para at� mesmo cobrar, com um argumento mais forte, seus diretos aos respons�veis.
+
+Para auxiliar no desenvolvimento do front-end da aplicação, utilizamos o framework Bootstrap, 
```

Comment:
... Bootstrap. Decidimos usá-lo para a fim de acelerar o processo de criação de uma interface web responsiva aos diferentes tamanhos de telas de dispositivos mobile.

* o início da oração seguinte sugeria acho que pode ficar para o próximo tópico que é a conclusão*